prepare_source

```diff
@@ -1,2 +1,31 @@
-apt_src ignition
+workdir=$(pwd)
+tmp_dir="$(mktemp -d)"
+trap 'cd / && rm -rf "$tmp_dir"' EXIT
+pushd "$tmp_dir"
+
+apt-get source --only-source --download-only ignition=2.14.0+ds1-1
```

Comment:
Have not checked in detail, but we may want to enhance [apt_src](https://github.com/gardenlinux/package-build/blob/96995631ca53a8293f4a8863eedd66a92ab09232/bin/source#L23) so it can download a specific version like you do here.

Reply:
Niko suggested a bit of a different way of doing things, which is definitely better, so using that.